src/zap/zapmetadata.cpp

```diff
@@ -129,10 +129,11 @@ int __cdecl RVAFieldCmp(const void * a_, const void * b_)
 
     if (a->pData != b->pData)
     {
+        // Ascending order on rva
         return (a->pData > b->pData) ? 1 : -1;
     }
 
-    return 0;
+    return (int)(b->cbSize - a->cbSize);    // Descending order on size
```

Comment:
It should not hurt to sort it based on size, but it should not be required for the rest of the algorithm in this file to work correctly.

Reply:
Yeah this more of a cosmetic change, but yes the `UpdateSizeAndAlignment` will take care of it. I can revert that line